Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Inn.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class Inn : MonoBehaviour
+{
+    public void HealTroops(Player player)
+    {
+		//Change magic number to function once more castle code is in
+		if (player.CastleProgress >= 4)
+			return;
+
+		List<Unit> units;
+		units = player.PlayerArmy.GetRandomUnits(3);
```

Comment:
This only needs to get units which are knocked out. No point healing alive ones.